server/internal/server/fixtures_test.go

```diff
@@ -0,0 +1,52 @@
+package server
```

Comment:
A better file name would be `test_fixtures.go` . How do you feel about making the functions public?
They would be still within the `internal` package and not visible when used as a library.

Reply:
Looks good! If we really want to make these work only in test we can change the functions to take `*testing.T` as a argument, but that would be probably overkill.